lib/db/schemaupdater.go

```diff
@@ -629,5 +670,275 @@ func (db *schemaUpdater) updateSchemaTo12(_ int) error {
 		return err
 	}
 
+	if err := db.rewriteGlobals(t); err != nil {
```

Comment:
I think we should make this a version 13 migration. Even though 12 is not out, people might be running master (I was...) and the effect then is that we skip this part of the migration now and checkGlobals deletes all the globals because they don't unmarshal correctly and then weirdness ensues.

Reply:
Done. And now doing the same - well, running ahead of master :)